Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/jvm/CommandLine.kt

```diff
@@ -64,11 +66,15 @@ class CInteropArguments : CommonInteropArguments() {
     @Argument(value = HEADER_FILTER_ADDITIONAL_SEARCH_PREFIX, shortName = "-hfasp",  valueDescription = "<file>", description = "header file to produce kotlin bindings for") 
     var headerFilterPrefix: Array<String> = arrayOf()
 
-    @Argument(value = "-compilerOpts", shortName = "-copt", valueDescription = "<arg>", description = "additional compiler options", delimiter = " ")
-    var compilerOpts: Array<String> = arrayOf()
+    @Argument(value = "-compilerOpts", shortName = "-copt", valueDescription = "<arg>", description = "additional compiler options", delimiter = ARGUMENT_NO_DELIMITER)
```

Comment:
This part interferes with #2672. On the other hand, it doesn't seem to fix anything from [KT-29970](
https://youtrack.jetbrains.com/issue/KT-29970).

Reply:
It fixes any possible escaped arguments here. Do you mean there are no such arguments possible?

Reply:
I mean that this part doesn't seem to fix anything from [KT-29970](https://youtrack.jetbrains.com/issue/KT-29970). It fixes an issue which you haven't encountered and reported yet, that's why it can be fixed later to avoid merge conflicts now.

Reply:
It is not totally true - https://youtrack.jetbrains.com/issue/KT-29970#focus=streamItem-27-3295557.0-0

Reply:
How does this affect you when using e.g. `A B C.framework`?

Reply:
It does not